# The command for the TOWR data generation

float32                goalX
float32                goalY
float32                goalYAW
float32                duration
bool                   end_opt                   # optimization end flag
